src/app/widgets/inlineinput/inlineinput.component.html

```diff
@@ -8,7 +8,7 @@
   [attr.disabled]="(!onLineClickEdit || readOnly) && !editing ? '' : null"
   [attr.placeholder]="placeholder"
   [class.active]="editing"
-  [innerHTML]="inpvalue"
+  [ngModel]="inpvalue"
```

Comment:
Apply camel case to inpvalue - inpValue or inputValue

Reply:
@sahil143 Special character does not render as expected after removing ```    innerHTML    ```.